packages/table/src/styledComponents/Table.js

```diff
@@ -16,6 +16,28 @@ const borderTop = props =>
     border-top: none;
   `;
 
+const stickyHeaderCss = ({ stickyHeader, stickyHeaderTopOffset }) =>
+  stickyHeader &&
+  css`
+    caption > span > div {
+      z-index: 3;
+      border-top: solid 1px ${colors.GREY_16};
+    }
+    thead .cell {
+      position: sticky;
+      top: ${stickyHeaderTopOffset}px;
+      z-index: 2;
+      border-bottom: solid 1px ${colors.GREY_16};
+      padding: 8px 0 8px 16px;
```

Comment:
This padding doesn't seem related to the sticky header.
I think we can apply it to all cells. Most of them have fixed heights anyway.